docs/tutorials/notes-mvvm/includes/viewmodel-about.md

```diff
@@ -0,0 +1,91 @@
+01. Right-click on the **ViewModels** folder > **Add** > **Class** and name it **AboutViewModel.cs**.
+01. Repeat the previous step and create two more view models:
+    - **NoteViewModel.cs**
+    - **NotesViewModel.cs**
+
+Your project structure should look like the following image:
+
+:::image type="content" source="../media/viewmodel-about/class-structure-1.png" alt-text="Solution explorer showing MVVM folders.":::
+
+## About viewmodel and About view
+
+The **About view** displays some data on the screen and optionally navigates to a website with more information. Since this view doesn't have any data to change, like with an entry control or selecting items from a list, it's a good candidate to demonstrate adding a viewmodel. For the **About viewmodel**, there isn't a backing model.
+
+Create the **About viewmodel**:
+
+01. In the **Solution Explorer** pane of Visual Studio, double-click on **ViewModels\\AboutViewModel.cs**.
+01. Paste in the following code:
+
+    :::code language="csharp" source="../snippets/viewmodel-shared/csharp/Notes/ViewModels/AboutViewModel.cs":::
+
+The previous code snippet contains some properties that represent information about the app, such as the name and version. This snippet is exactly the same as the **About model** you deleted earlier. However, this viewmodel contains a new concept, the `ShowMoreInfoCommand` command property.
+
+Commands are bindable actions that invoke code, and are a great place to put app logic. In this example, the `ShowMoreInfoCommand` points to the `ShowMoreInfo` method, which opens the web browser to a specific page. You'll learn more about the command system in the next section.
+
+### About view
+
+The **About view** needs to be changed slightly to hook it up to the viewmodel that was created in the previous section. In the _Views\\AboutPage.xaml_ file,  file, apply the following changes:
+
+- Update the `xmlns:models` XML namespace to `xmlns:viewModels` and target the `Notes.ViewModels` .NET namespace.
+- Change the `ContentPage.BindingContext` property to a new instance of the `About` viewmodel.
+- Remove the button's `Clicked` event handler and use the `Command` property.
+
+Update the **About view**:
+
+01. In the **Solution Explorer** pane of Visual Studio, double-click on **Views\\AboutPage.xaml**.
+01. Paste in the following code:
+
+    :::code language="xaml" source="../snippets/viewmodel-shared/csharp/Notes/Views/AboutPage.xaml" highlight="4,7,19":::
+
+    The previous code snippet highlights the lines that have changed in this version of the view.
+
+Notice that the button is using the `Command` property. Many controls have a `Command` property that is invoked when the user interacts with the control. When used with a button, the command is invoked when a user presses the button, similar to how the `Clicked` event handler is invoked, except that you can bind `Command` to the viewmodel.
```

Comment:
"except that you can bind `Command` to the viewmodel" > "except that you can bind `Command` to a property in the viewmodel."?